pkg/ctl/install/install.go

```diff
@@ -0,0 +1,435 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/justinbarrick/go-k8s-portforward"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/riywo/loginshell"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	fluxapi "github.com/weaveworks/flux/api/v6"
+	transport "github.com/weaveworks/flux/http"
+	"github.com/weaveworks/flux/http/client"
+	"github.com/weaveworks/flux/install"
+	"github.com/weaveworks/flux/ssh"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/utils/kubeconfig"
+)
+
+const namespaceFileName = "namespace.yaml"
+
+// Command will create the `install` commands
+func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
+	verbCmd := cmdutils.NewVerbCmd("install", "Install components in a cluster", "")
+
+	cmdutils.AddResourceCmd(flagGrouping, verbCmd, installFluxCmd)
+
+	return verbCmd
+}
+
+type installFluxOpts struct {
+	templateParams install.TemplateParameters
+	gitFluxPath    string
+	timeout        time.Duration
+	amend          bool
+}
+
+func installFluxCmd(rc *cmdutils.ResourceCmd) {
+	rc.ClusterConfig = api.NewClusterConfig()
+	rc.SetDescription(
+		"flux",
+		"Bootstrap Flux, installing it in the cluster and initializing its manifests in the Git repository",
+		"",
+		)
+	var opts installFluxOpts
+	rc.SetRunFuncWithNameArg(func() error {
+		installer, err := newFluxInstaller(rc, &opts)
+		if err != nil {
+			return err
+		}
+		return installer.run(context.Background())
+	})
+
+	rc.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
```

Comment:
```suggestion
	rc.FlagSetGroup.InFlagSet("Flux", func(fs *pflag.FlagSet) {
```

That will appear as `Flux Flags` in help output.

Reply:
**Before**

```console
$ EKSCTL_EXPERIMENTAL=true ./eksctl help install flux
Bootstrap Flux, installing it in the cluster and initializing its manifests in the Git repository

Usage: eksctl install flux [flags]

General flags:
      --git-url string           URL of the Git repository to be used by Flux, e.g. git@github.com:<your username>/flux-get-started
      --git-branch string        Git branch to be used by Flux (default "master")
      --git-paths strings        Relative paths within the Git repo for Flux to locate Kubernetes manifests
      --git-label string         Git label to keep track of Flux's sync progress; overrides both --git-sync-tag and --git-notes-ref (default "flux")
      --git-user string          Username to use as Git committer (default "Flux")
      --git-email string         Email to use as Git committer
      --git-flux-subdir string   Directory within the Git repository where to commit the Flux manifests (default "flux/")
      --timeout duration         Timeout duration for I/O operations (default 20s)
      --namespace string         Cluster namespace where to install Flux (default "flux")
      --amend                    Stop to manually tweak the Flux manifests before pushing them to the Git repository
  -n, --name string              EKS cluster name
  -r, --region string            AWS region
  -f, --config-file string       load configuration from a file (or stdin if set to '-')

Common flags:
  -C, --color string   toggle colorized logs (valid options: true, false, fabulous) (default "true")
  -h, --help           help for this command
  -v, --verbose int    set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging (default 3)

Use 'eksctl install flux [command] --help' for more information about a command.
```

**After**:

```console
$ EKSCTL_EXPERIMENTAL=true ./eksctl help install flux
Bootstrap Flux, installing it in the cluster and initializing its manifests in the Git repository

Usage: eksctl install flux [flags]

Flux installation flags:
      --git-url string           URL of the Git repository to be used by Flux, e.g. git@github.com:<your username>/flux-get-started
      --git-branch string        Git branch to be used by Flux (default "master")
      --git-paths strings        Relative paths within the Git repo for Flux to locate Kubernetes manifests
      --git-label string         Git label to keep track of Flux's sync progress; overrides both --git-sync-tag and --git-notes-ref (default "flux")
      --git-user string          Username to use as Git committer (default "Flux")
      --git-email string         Email to use as Git committer
      --git-flux-subdir string   Directory within the Git repository where to commit the Flux manifests (default "flux/")
      --timeout duration         Timeout for I/O operations (default 20s)
      --namespace string         Cluster namespace where to install Flux (default "flux")
      --amend                    Stop to manually tweak the Flux manifests before pushing them to the Git repository

General flags:
  -n, --name string          EKS cluster name
  -r, --region string        AWS region
  -f, --config-file string   load configuration from a file (or stdin if set to '-')

Common flags:
  -C, --color string   toggle colorized logs (valid options: true, false, fabulous) (default "true")
  -h, --help           help for this command
  -v, --verbose int    set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging (default 3)

Use 'eksctl install flux [command] --help' for more information about a command.
```